python/plugins/grassprovider/description/r.path.txt

```diff
@@ -0,0 +1,13 @@
+r.path
+Traces paths from starting points following input directions.
+Raster (r.*)
+QgsProcessingParameterRasterLayer|input|Name of input direction
+QgsProcessingParameterEnum|format|Format of the input direction map|auto;degree;45degree;bitmask|false|0|false
+QgsProcessingParameterRasterLayer|values|Name of input raster values to be used for output|None|True
+QgsProcessingParameterRasterDestination|raster_path|Name for output raster path map
+QgsProcessingParameterVectorDestination|vector_path|Name for output vector path map
+QgsProcessingParameterPoint|start_coordinates|Map coordinates of starting point(s) (E,N)|None|True
+QgsProcessingParameterFeatureSource|start_points|Vector layer containing starting point(s)|0|None|True
```

Comment:
Better to set allowed geometry type to points.

Also probably it is worth to split the tool into two: one accepting point and another one accepting vector layer with starting points?

Reply:
@alexbruy 



> Better to set allowed geometry type to points.

isn't 0 already correct for points? It does not list lines or polygons here with 0.

Reply:
> Also probably it is worth to split the tool into two: one accepting point and another one accepting vector layer with starting points?

done